README.rst

```diff
@@ -1,19 +1,111 @@
+####################
 edx-name-affirmation
+####################
+
+| |Build Status| |Coveralls|
+
+This library contains data model and business logic to store verified 
+personal information for learners of the edx-platform.
+This library is designed to be a pluggable library to the edx-platform.
+
+Overview
+========
+
+This library mainly stores learner's various stages of a learner's verified identifying information.
+The prevalent example is the learner's full name. To verify the full name, the learner
+has to first initate from UI that is part of the platform.
+Here are the steps of which the PII information, for example: full name, goes through:
+
+1. learner requests to update the name
+2. learner is brought to go through ID verification process
+3. learner submit the ID verification after providing supporting evidence
+4. the ID Verification reviewer approves or denies the verification
+5. the library receives the verdit and updates the name record in the model accordingly
+
+The above process can also be retriggered by a Proctored exam attempt, where the exam attempt status
+can be the input into this library to create and update the name record's status.
+
+For more context of the library, see `context`_.
+
+Installing in Docker Devstack
+-----------------------------
+
+Assuming that your ``devstack`` repo lives at ``~/edx/devstack``
```

Comment:
This is only if you want an editable installation, right? `edx-name-affirmation` should be installed into edx-platform in devstack by default. If so, we should call it out so that people don't do extra work.

Does adding `-e /edx/src/edx-name-affirmation` to the `private.txt` file in edx-platform work? That's what we do for `edx-proctoring`, and it's nice, because you don't have to follow the below steps each time you need to install requirements.

Reply:
Add to `private.txt` works. However, that require the developer run pip install again, which is time costly. I can say add to `private.txt` as an alternative option tho.